docs/inbound-webhook.md

```
@@ -0,0 +1,220 @@
+python3 -c "import secrets; print(secrets.token_urlsafe(32))"
+```
+
+### 2. Store it in SWH settings
+
+1. Go to **Tickets → Settings → Integrations**.
+2. Paste your secret into **Webhook Secret**.
+3. Save.
+
+The endpoint URL is shown on the same page:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)